spec/haskell/src/SEL4/Kernel/Thread.lhs

```diff
@@ -854,5 +854,3 @@ Kernel init will created a initial thread whose tcbPriority is max priority.
 >         new_queue <- tcbQueueRemove releaseQueue tcbPtr
 >         setReleaseQueue new_queue
 >         threadSet (\t -> t { tcbInReleaseQueue = False }) tcbPtr
-
-%
```

Comment:
Looks like accidentally committed, should be removed from this commit.

Reply:
I tried to revert this change, but I'm not sure it worked. I always get confused about a blank line at the end and what the red sign thing means

Reply:
OK I think I've got it now